app/javascript/flavours/glitch/styles/components/status.scss

```diff
@@ -590,6 +590,10 @@
   width: 23.15px;
 }
 
+.status__action-bar-timestamp {
+  margin: 0 0 0 auto;
```

Comment:
Maybe:
```suggestion
  flex-grow: 1;
  text-align: end;
```

Reply:
That works. Could you explain why that's preferred (or maybe my CSS know-how is just outdated idk)?

Reply:
RTL languages!